config/paging_schedule.rb

```diff
@@ -0,0 +1,105 @@
+PagingSchedule.configure do
```

Comment:
I'm not clear why `config/paging_schedule` and `config/initializers/paging_schedule` are both needed. I'd naively expect this content in the initializer.